controllers/upgrade.go

```diff
@@ -113,6 +121,44 @@ func (r *RollingUpgradeReconciler) ReplaceNodeBatch(rollingUpgrade *v1alpha1.Rol
 	return true, nil
 }
 
+func (r *RollingUpgradeReconciler) SetInstanceTag(rollingUpgrade *v1alpha1.RollingUpgrade, instance *autoscaling.Instance, tagKey string, tagVal string) error {
+	return awsprovider.TagEC2instance(aws.StringValue(instance.InstanceId), tagKey, tagVal, r.Auth.Ec2Client)
```

Comment:
Once you move `TagEC2instance` under the `AmazonClientSet` this call should change to be made through that object